VTEX - Logistics API.json

```diff
@@ -5463,7 +5454,7 @@
                                                             "items": {
                                                                 "type": "string"
                                                             },
-                                                            "description": ""
+                                                            "description": "[Kit](https://help.vtex.com/en/tutorial/what-is-a-kit--5ov5s3eHM4AqAAgqWwoc28) item details."
```

Comment:
Verificar se será [Kit] ou [Kit item] neste tipo de descrição.
As demais descrições do PR estão como [Kit item].

Reply:
Boa, padronizei por "[Kit](https://help.vtex.com/en/tutorial/what-is-a-kit--5ov5s3eHM4AqAAgqWwoc28) item information."